### PR TITLE
Create related-projects.md

### PR DESCRIPTION
**Pull request by fdietze**
I think we should collect more insights about related projects. Please add your own projects and insights you have about other projects.

Of course the table columns can be changed and extended.

### CODE REVIEW

File: related-projects.md [hunk 1]
```diff
@@ -1,7 +1,36 @@
+
 # Overview Related Projects
 
 Please add your own projects and insights you have about others.
 
-| Name/Link | Description | Interesting ideas |
-| --- | --- | --- |
-| [Concordis](https://concordis.io/) |  | <ul><li>Re-Use twitter identity verification</li><li>Challenge thought leaders with argument maps on twitter.</li></ul> |
+| Name/Link | Description |
```
Comment (fdietze) [inline related-projects.md:6]: Why did you delete the `Interesting Ideas` column?

Comment (Whathecode) [reply related-projects.md:6]: As mentioned on Slack: "I removed the 'interesting insights' column as it is way too subjective. Won't be maintainable. Instead,  I feel that should go in the description column."
